zuul.d/projects.yaml

```diff
@@ -471,3 +471,41 @@
     default-branch: main
     templates:
       - publish-to-automation-hub
+
+- project:
+    name: ansible-collections/ansible_hub
+    default-branch: main
+    templates:
+      - publish-to-galaxy
+      - publish-to-automation-hub
+
+- project:
+    name: ansible-collections/ansible/event-driven-ansible
+    default-branch: main
+    templates:
+      - publish-to-galaxy
+      - publish-to-automation-hub
```

Comment:
We probably only need `publish-to-automation-hub`.  cc @Alex-Izquierdo

Reply:
That's correct. Our GH workflow already publishes to galaxy.